src/main/webapp/WEB-INF/tags/student/comments/feedbackQuestionPanel.tag

```diff
@@ -0,0 +1,21 @@
+<%@ tag description="Feedback question panel" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/student/comments" prefix="comments" %>
+<%@ attribute name="feedbackQuestionTable" type="teammates.ui.template.QuestionTable" required="true" %>
+<%@ attribute name="fsIdx" required="true" %>
+<%@ attribute name="qnIdx" required="true" %>
+<div class="panel panel-info">
+    <div class="panel-heading">
+        <b>Question ${feedbackQuestionTable.questionNumber}</b>:
+        ${feedbackQuestionTable.questionText}
+    </div>
+    <table class="table">
+        <tbody>
+            <c:forEach items="${feedbackQuestionTable.responseRows}" var="feedbackResponseRow" varStatus="j"> <%-- FeedbackResponse loop starts --%>
+                <c:set var="responseIndex" value="${j.index + 1}" />
+                <comments:feedbackResponsePanel feedbackResponseRow="${feedbackResponseRow}"
+                 fsIdx="${fsIdx}" qnIdx="${qnIdx}" responseIndex="${responseIndex}"/>
```

Comment:
Wouldn't it be easier passing in `${j.index + 1}` without setting it into `${responseIndex}`?
There's the `responseIndex=` part, so it won't lose clarity or anything.
Issues similar to this happens elsewhere too.